cave_sea/map.conf

```
@@ -0,0 +1,18 @@
+team.2.color = blue
+team.1.pos = (-7,2,6)
+team.2 = blue
+team.1.color = red
+team.1 = red
+team.2.pos = (-7,2,-6)
+h = 150
+r = 75
+rotation = z
+author = sodomit
+name = Cave Sea
+base_node = ctf_map:glass
+initial_stuff = grenades:frag 1,default:mese_post_light 20
+chests.1.n = 300
+chests.1.from = -75,-75,-75
+chests.1.to = 75,75,75
+treasures = default:pick_diamond,1,0,10;shooter:ammo,1,2,999;shooter:shotgun,1,10,1;shooter:machine_gun,1,10,1;grenades:frag,1,10,999;sniper_rifles:rifle_magnum,1,10,1
```

```suggestion
treasures = default:pick_diamond,1,0,10;shooter:ammo,1,2,5;shooter:shotgun,1,10,1;shooter:machine_gun,1,10,1;grenades:frag,1,10;sniper_rifles:rifle_762,1,10,1
```

I think shotguns and smgs could be OP on this map, especially because there are a lot of tunnels so one can easily kill you.
Maybe adding some more items to custom treasure wouldn't be a bad idea.